exercises/alphametics/example.rb

```diff
@@ -0,0 +1,37 @@
+class Alphametics
+  VERSION = 1
+
+  def solve(puzzle)
+    letters = Hash.new(0)
+    puzzle = format_to_ruby(puzzle)
+    puzzle.scan(/[a-zA-Z]/) { |w| letters[w] += 1 }
+
+    possible_values(letters.keys) do |letters_values|
+      return letters_values if valid?(puzzle, letters_values)
+    end
+
+    nil
+  end
+
+  private
+
+  def format_to_ruby(puzzle)
+    puzzle.gsub(/\^/, '**') # format exponent: 2 ^ 3 to 2 ** 3
```

Comment:
No, but also I would not put spaces around the `**`.  It looks strange, there probably shouldn't have been spaces around the original exponent symbol.  It isn't how we do it on paper, it should be the same, if possible in the language, in code.